octoprint_Spoolman/common/settings.py

```diff
@@ -8,3 +8,4 @@ class SettingsKeys():
 	SPOOLMAN_CERT_PEM_PATH = "spoolmanCertPemPath"
 	SELECTED_SPOOL_IDS = "selectedSpoolIds"
 	IS_PREPRINT_SPOOL_VERIFY_ENABLED = "isPreprintSpoolVerifyEnabled"
+	DATA_SAVE = "dataSave"
```

Comment:
Please choose a more descriptive name for this property. Right now it does not reflect its purpose at all. Ideally, it should be tied to the fact that this is a backup feature which persists temporary data in the background.

Reply:
Also, this feature needs to be toggleable via settings. Even with Octoprint 1.10.0 which significantly improves Settings performance, changing any setting can still be a bit taxing on lower-spec machines. For people with such Octoprint machines, it should be possible to disable this feature if it introduces too much of a performance hit.